src/k2/dto/MessageVerbs.h

```diff
@@ -70,7 +75,7 @@ enum Verbs : k2::Verb {
     PERSISTENT_APPEND,
     PERSISTENT_READ,
     PERSISTENT_SEAL,
-
+    PERSISTENT_STATUS,
```

Comment:
I think I don't quite like these names. I think we should go with PLOG_* to make it clear. For status, please use the name PLOG_GET_STATUS to express an action/verb

Reply:
changed